WooCommerce/Classes/Authentication/Epilogue/SwitchStoreNoticePresenter.swift

```diff
@@ -24,13 +27,25 @@ final class SwitchStoreNoticePresenter {
             return
         }
 
-        let messageFormat = NSLocalizedString("Switched to %1$@. You will only receive notifications from this store.",
-                                              comment: "Message presented after users switch to a new store. "
-                                                + "Reads like: Switched to {store name}. You will only receive notifications from this store. "
-                                                + "Parameters: %1$@ - store name")
-        let message = String.localizedStringWithFormat(messageFormat, newStoreName)
-        let notice = Notice(title: message, feedbackType: .success)
+        let titleFormat = featureFlagService.isFeatureFlagEnabled(.pushNotificationsForAllStores) ?
+            Localization.titleFormat: Localization.titleFormatWithPushNotificationsForAllStoresDisabled
+        let title = String.localizedStringWithFormat(titleFormat, newStoreName)
+        let notice = Notice(title: title, feedbackType: .success)
 
         noticePresenter.enqueue(notice: notice)
     }
 }
+
+extension SwitchStoreNoticePresenter {
+    enum Localization {
+        static let titleFormatWithPushNotificationsForAllStoresDisabled =
+            NSLocalizedString("Switched to %1$@. You will only receive notifications from this store.",
+                              comment: "Message presented after users switch to a new store when multi-store push notifications are not supported. "
+                                + "Reads like: Switched to {store name}. You will only receive notifications from this store. "
+                                + "Parameters: %1$@ - store name")
+        static let titleFormat = NSLocalizedString("Switched to %1$@.",
+                                                   comment: "Message presented after users switch to a new store. "
+                                                    + "Reads like: Switched to {store name}. You will only receive notifications from this store. "
```

Comment:
Nit: this comment should be `Reads like: Switched to {store name}.` only right? 😅

Reply:
Great catch! Fixed in https://github.com/woocommerce/woocommerce-ios/pull/5137/commits/15981818fd0b1004434fda23708f380d619ac7ab